src/leetCodeExecutor.ts

```diff
@@ -11,14 +11,17 @@ import { executeCommand, executeCommandWithProgress } from "./utils/cpUtils";
 import { genFileName } from "./utils/problemUtils";
 import { DialogOptions, openUrl } from "./utils/uiUtils";
 import * as wsl from "./utils/wslUtils";
+import { toWslPath, useWsl } from "./utils/wslUtils";
 
 class LeetCodeExecutor {
     private leetCodeRootPath: string;
     private leetCodeRootPathInWsl: string;
+    private nodeExecutable: string;
 
     constructor() {
         this.leetCodeRootPath = path.join(__dirname, "..", "..", "node_modules", "vsc-leetcode-cli");
         this.leetCodeRootPathInWsl = "";
+        this.nodeExecutable = this.getNodePath();
```

Comment:
Currently, `this.nodeExecutable` is set only once in intialization. If the user changes the `nodePath` config, it won't take effect until reloading. 

Maybe `this.nodeExecutable` could be set as a getter, or set a listener on configuration changed.

Reply:
Add a listener to listen to the configuration change is a good idea. Will update the code.